### ISSUE TITLE
a11y issue - OverflowMenu component's dropdown is not accessible using tabs or down arrow keys.

### ISSUE THREAD

**Issue by miramurali23** [carbon-design-system/carbon-components-react#557]
Hey ya'll - working on improving accessibility and the Overflow Menu is accessible if you physically click on it with your mouse - but for Screen Reader users they have to use tabs, spaces or the arrow keys...the content in the Overflow menu is unable to be accessed. 

<img width="1430" alt="screen shot 2018-01-30 at 2 46 46 pm" src="https://user-images.githubusercontent.com/11800028/35590476-a68115d4-05cc-11e8-9b1c-6b9cabd0e675.png">

^^unable to access Stop and Reboot functionality if I am using tab, spaces or down arrow key.

**Comment by joshblack** [comment]
Hi @miramurali23! 👋  Thanks so much for taking the time to make this issue 😄  and thanks so much for suggesting an a11y improvement. Will definitely try and get this prioritized and implemented, and also will accept any Pull Requests that help out with this!

**Comment by tw15egan** [comment]
Possibly related to: https://github.com/carbon-design-system/carbon-components-react/issues/557
